datalad/distribution/tests/test_dataset.py

```diff
@@ -194,3 +195,24 @@ def test_subdatasets(path):
     assert_true(ds.is_installed())
     eq_(ds.get_subdatasets(), [])
     # TODO actual submodule checkout is still there
+
+
+@with_tempfile(mkdir=True)
+def test_require_dataset(path):
+    # in this folder by default
+    assert_equal(
+        require_dataset(None).path,
+        abspath(os.path.curdir))
```

Comment:
@mih so this test requires  current directory to be proper initialized dataset?? imho not quite kosher, e.g. in my case my workdir git repo is not considered such -- not sure yet what is the culprit but that is the case.  Just wondered if I should fix this test to run the check when knowing for sure that curdir is a dataset or look into addressing why my git repo is not a good enough